docs/language_specification/about_the_grammar_sections.md

```diff
@@ -0,0 +1,14 @@
+1. In the syntax notation used in this documentation, syntactic categories are indicated by _italic_ type,
+and literal words and characters in <code>__boldface constant width__</code> type.</br></br>
```

Comment:
But the point is that it adds clarity to the developer, and it doesn't affect the output.

Reply:
I see, good point. For checking the changes in git it is indeed easier; so let's indeed break the sentences. 

I've seen it done in other projects both ways, even in the same file sometimes :(, that's what we definitely need to avoid.